### PR TITLE
feat: adopt Maeda's Code of Conduct as org default

### LINKED ISSUE #1: Repo defaults at the org level
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

### LINKED ISSUE #2: Establish org Code of Conduct
Establish a shared code of conduct for all repos in the PublicisSapient GitHub org.

This should use the .github repo established for #1

Initial code of conduct should probably adopt those already drafted in [PublicisSapient/ccoc](https://github.com/PublicisSapient/ccoc)

### PR DESCRIPTION
**Pull request by developher-net**
Establishes a default code of conduct for all repos in the PS GitHub org.
.github repo gets auto-inherted by all repos

Fixes #1
Fixes #2

//TODO - Archive the CCOC repo as this would supersede it

Original: https://github.com/PublicisSapient/ccoc

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Communications Code of Conduct (CCOC)
+# Community Code of Conduct (CCOC)
```
Comment (cssmiles) [inline CODE_OF_CONDUCT.md:1]: Typically open source projects adapts [code of conduct](https://www.contributor-covenant.org/version/2/0/code_of_conduct/code_of_conduct.md) from  [contributor covenant](https://www.contributor-covenant.org/). What's your thought on following a similar pattern and building on top of that. ​
​Also,  Naresh and I worked on baseline standards for one of the [repo](https://github.com/PublicisSapient/react-accelerate-modules) around contributing guidelines, RFCs, etc. and perhaps we can pick some work from there for this baseline repo.

Comment (developher-net) [reply CODE_OF_CONDUCT.md:1]: @cssmiles love the idea of reusing. And we definitely should look at everything we have for each standard we introduce at the org level.

The Contributor Convenant's code of conduct is one of two templates GitHub provides a template for. The other is a "Citizen's Code of Conduct" that is more for events, and not really appropriate for what we're doing here.

I like the one in this PR as it is derived from PS' core values. But I'm equally open to using Contributor Convenant as our starting point.

@nareshbhatia @iamdavidjackson your thoughts?

Comment (iamdavidjackson) [reply CODE_OF_CONDUCT.md:1]: I agree with you Anthony.  I like that it is based on our core values too. I think this is a great place to start.

**Review by iamdavidjackson — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (iamdavidjackson) [reply CODE_OF_CONDUCT.md:1]: Are we able to add a default contribution markdown as CSP suggested?  I think that would be valuable to have as it's easy to forget to add those.

Comment (developher-net) [reply CODE_OF_CONDUCT.md:1]: @iamdavidjackson absolutely - we have that as a task in the project as well. Now that we have this repo we might want to consider making that public so it's easier to find

**Pull request merged**